scripts/proxygen/deploy.py

```diff
@@ -0,0 +1,10 @@
+import fire
+
+
+def deploy_proxy(env: str, app_alias="dev"):
+
+    return "Not implemented yet"
```

Comment:
```suggestion
    raise NotImplementedError
```

Reply:
Yeah ill delete the extra proxygen bits - didnt finish it